apps/prairielearn/src/sync/course-db.ts

```diff
@@ -559,17 +518,22 @@ export function courseDataHasErrorsOrWarnings(courseData) {
  * Loads a JSON file at the path `path.join(coursePath, filePath). The
  * path is passed as two separate paths so that we can avoid leaking the
  * absolute path on disk to users.
- * @template {{ uuid: string }} T
- * @param {Object} options - Options for loading and validating the file
- * @param {string} options.coursePath
- * @param {string} options.filePath
- * @param {object} [options.schema]
+ * @param {object} options
```

Comment:
This and the below line contain unneeded type declarations; remove in favor of native TypeScript types.